feat: add findDidByName examples for nodejs and python

3kz2501 opened the pull request:
Add example scripts to demonstrate finding device identifiers by name using the /identifiers endpoint with device_name query parameter

vaachii commented on README.md:
````diff
@@ -42,6 +42,126 @@ By abstracting every device and cloud as globally unique endpoints and building
+  console.log(decrypted); //gives the original text i.e. "hello"
+```
+
+  Further Info:
````
@chivalryytf

Too much detailed information is incompatible with keeping the README clean, so move it to the Wiki or official documentation later.

chivalryytf replied:
Done!

vaachii commented on src/cipher/cipher.rs:
```diff
@@ -0,0 +1,140 @@
+
+    
+    return result;
+  }
```
@chivalryytf 

Please add Rust-style unit tests. Please do the same for the `Hasher` and `Signer` as well as the `Cipher`. Because what we provide as UNiD Edge SDK is not a NodeJS-style library, but essentially a Rust-style library. As for the test cases, please port the ones implemented as NodeJS-style unit tests to Rust-style unit tests.

chivalryytf replied:
Cipher unit testing is done! Remaining Hasher and Signer!

vaachii reviewed (commented):
@chivalryytf

I was reviewed a PR! Please check it.

vaachii commented:
@chivalryytf

Thank you commit @chivalryytf !

3kz2501 committed:
feat: add findDidByName examples for nodejs and python

Add example scripts to demonstrate finding device identifiers by name
using the /identifiers endpoint with device_name query parameter
